.actrc

```diff
@@ -0,0 +1 @@
+-P ubuntu-latest=nektos/act-environments-ubuntu:18.04
```

Comment:
I'm going to add documentation on running `act` to test Github workflows - but this will ensure that when running the Action, you're using the same image used for running locally that is run remotely.